README.md

```diff
@@ -9,12 +9,17 @@
 ## Usage
 
 1. Install the app on your GitHub Repositories: [github.com/apps/wip](https://github.com/apps/wip)
-2. The WIP bot sets status of the request title to pending if it finds  "wip", "work in progress" or "do not merge" (not case-sensitive) in
+2. If the WIP bot finds "wip", "work in progresss" or "do not merge" (not case-sensitive) in
```

Comment:
Typo: 3 s in progress

Reply:
Bah. I do this _all the time_. Thanks/sorry.